db/v-2/tables/high-level-names.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="add-high-level-names-table" author="sanyavertolet" context="dev or prod">
+        <createTable tableName="high_level_names">
+            <column name="name" type="varchar(64)">
```

Comment:
what's about foreign key to organization, user and contest?

Reply:
There is no need to do it for contest
But as for organization and user, I thought it might be less convenient to rename organizations and users